packages/ui/src/components/vuestic-components/va-rating/VaRatingItem.vue

```diff
@@ -1,116 +1,134 @@
 <template>
   <div
-    class="va-rating-item"
-    @keyup.enter="onEnter()"
     @mousemove="onHover"
-    @click="onClick()"
     @mouseleave="removeHover"
+    @focus="onFocus"
```

Comment:
It's better to close brackets if no params are being passed.
I.e. in this case `onFocus()` instead of `onFocus`. Tells clearly no additional processing is happening with event.